package.json

```diff
@@ -11,31 +11,37 @@
     "jest": "jest --config ./jest.config.js"
   },
   "dependencies": {
-    "@types/node": "^11.9.6",
-    "@types/react-redux": "^7.0.1",
-    "@types/webpack-env": "^1.13.7",
-    "expo": "^32.0.0",
+    "@types/node": "11.9.6",
+    "@types/react-redux": "7.0.1",
+    "@types/webpack-env": "1.13.7",
+    "expo": "32.0.0",
     "react": "16.5.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
-    "react-native-debugger-open": "^0.3.18",
-    "react-redux": "^6.0.1",
-    "redux": "^4.0.1",
-    "redux-saga": "^1.0.2"
+    "react-native-debugger-open": "0.3.18",
+    "react-redux": "6.0.1",
+    "redux": "4.0.1",
+    "redux-saga": "1.0.2"
   },
   "devDependencies": {
-    "@types/expo": "^32.0.3",
-    "@types/jest": "^24.0.9",
-    "@types/react": "^16.8.6",
-    "@types/react-test-renderer": "^16.8.1",
-    "jest": "^24.1.0",
-    "jest-expo": "^32.0.0",
-    "react-test-renderer": "^16.8.3",
-    "redux-devtools-extension": "^2.13.8",
```

Comment:
redux-devtools-extension -- we need it. Why is it removed?

Reply:
![image](https://user-images.githubusercontent.com/14149177/53869645-a940f280-4009-11e9-8500-055a452fcf61.png)